tests/functional/ca/meson.build

```diff
@@ -24,7 +24,7 @@ suites += {
     'nix-shell.sh',
     'post-hook.sh',
     'recursive.sh',
-    'repl.sh',
+    # 'repl.sh',  # Temporarily disabled - failing test
```

Comment:
Still needed?

Reply:
Those two fail locally for me still. I will drop the commit that comments them out before merging.